st2tests/st2tests/fixtures/generic/runners/actionchain.yaml

```diff
@@ -4,3 +4,6 @@ enabled: true
 name: action-chain
 runner_module: action_chain_runner
 runner_parameters: {}
+output_schema:
```

Comment:
Again, I prefer the absence of an output_schema over explicitly saying unmodeled here.  Also, even if I agree, this is the same problem with output_path, i prefer the unmodeled to be outside of the output_schema so not to be confused with the actual parameters.

Reply:
Agree. I've added that in most recent commit.